.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: "2.1"
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      - lint
+      - build
+      - test
```

Comment:
Eventually we might want a pre-step to cache dependencies. If we hit that point, we'll need to deal with golang/go#43994